### DISCUSSION TITLE
Change url

### DISCUSSION THREAD

**Discussion by hamechiz** [LiveHelperChat/livehelperchat#1815] [General]
Hello

I wanted to know which part I should go to after installation to change the address

Thanks

**Comment by PeopleInside** [comment]
Hi, what do you mean with "I wanted to know which part I should go to after installation to change the address" ?
What address? How you wanna change? Different domains?
Remove a part of the URL?

If you could be more specific maybe will be more easy answer to you.

**Comment by hamechiz** [comment]
Hello
Yes, please tell all the items
Thanks

**Reply by remdex** [reply]
https://doc.livehelperchat.com/docs/development/move-chat
